properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -213,7 +213,7 @@ making sure that you maintain a proper JSON format.
 								"title": "PROPERTIES_PANE___DIRECTIVES_ARGUMENTS_INFO_MODAL_TITLE",
 								"content": "PROPERTIES_PANE___DIRECTIVES_ARGUMENTS_INFO_MODAL_BODY",
 								"featureNameForHelp": "GraphQL directives",
```

Comment:
We should probably remove `featureNameForHelp` too.
It was used in the email template for "contact support".